.github/workflows/publish_docker.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         env:
           DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
           DOCKER_PASS: ${{ secrets.DOCKER_PASS }}
-          IMAGE_NAME: "avaplatform/subnet-evm"
+          IMAGE_NAME: "avaplatform/avalanchego-subnet-evm"
```

Comment:
pretty sure we rely on this in much of the infra, better check with infra first.

Reply:
we have checked this with infra, and seems there is no blocker as they manage their own images.